projects/pastanaga/src/lib/badge/badge.component.spec.ts

```diff
@@ -7,10 +7,10 @@ import {
 } from '../..';
 import { CommonModule } from '@angular/common';
 import { Component } from '@angular/core';
-import { TranslateLoader, TranslateModule } from '@ngx-translate/core';
 import { AngularSvgIconModule, SvgLoader } from 'angular-svg-icon';
-import { FakeTranslateLoader, svgLoaderFactory } from '../test.utils';
-
+import { svgLoaderFactory } from '../test.utils';
+import { TranslateModule } from '../translate/translate.module';
+import * as en from '../../../../../src/assets/i18n/en.json';
```

Comment:
Hum, I would give a specific test translation file, which should be next to the test file.